generators/server/templates/src/test/java/package/web/rest/errors/_ExceptionTranslatorIntTest.java

```diff
@@ -91,9 +91,8 @@ public void testMethodArgumentNotValid() throws Exception {
              .andExpect(status().isBadRequest())
              .andExpect(content().contentType(MediaTypes.PROBLEM))
              .andExpect(jsonPath("$.message").value(ErrorConstants.ERR_VALIDATION))
-             .andExpect(jsonPath("$.fieldErrors.[0].objectName").value("testDTO"))
```

Comment:
This was used in i18n and I dont see the benefit of this PR if it removes existing functionality